Dockerfile

```diff
@@ -3,10 +3,11 @@ FROM python:3.8-slim
 RUN mkdir -p /app
 WORKDIR /app
 
-COPY requirements.txt requirements.txt
-RUN pip install -r requirements.txt
+RUN apt-get update && apt-get install -y git
 
 COPY . .
```

Comment:
Also copies `requirements.txt`, so we can drop `COPY requirements.txt requirements.txt`